NMV_translations.json

```diff
@@ -0,0 +1 @@
","utter","adduce","declare","tongue","intimate","rehearse","bubble","cite","mouth","observe","relate","inform"]],["«این",["this","which"]],["است",["be","stand","exist"]],["اکنون",["now","here","present"]],["استخوانی",["bony","osseous","osteoid"]],["از",["of","from","in","by","greed","avarice","avidity","esurience","rapacity","ab-","abs-"]],["استخوانهایم،",["bone"]],["و",["and"]],["گوشتی",["meaty","fleshy"]],["از",["of","from","in","by","greed","avarice","avidity","esurience","rapacity","ab-","abs-"]],["گوشتم؛",["meat","flesh","bully","brawn","viand"]],["او",["he","she","it"]],["زن",["woman","female","wife","distaff","carline"]],["نامیده",["yclept"]],["شود،",["dill"]],["زیرا",["because","owing to"]],["که",["that","than","as if","which","who","whom"]],["از",["of","from","in","by","greed","avarice","avidity","esurience","rapacity","ab-","abs-"]],["مرد",["man","guy","husband","fellow","chap","groom","playmate","gink","hidalgo","manly","valiant","brave"]],["گرفته",["solemn","gruff","eerie","hoarse","dull","muzzy","darksome","thick","choky","eery","muggy","pokey","poky","sombrous","tristful","chock-full","low-spirited"]],["شد.»",["become","go","grow","be","happen","leave"]]],[["از",["of","from","in","by","greed","avarice","avidity","esurience","rapacity","ab-","abs-"]],["همین",["That's it."]],["رو،",["face","top","superficies","visage","uppermost"]],["مرد",["man","guy","husband","fellow","chap","groom","playmate","gink","hidalgo","manly","valiant","brave"]],["پدر",["father","dad","sire","pater","abba","baba"]],["و",["and"]],["مادر",["mother","mum","mama","mater","matriarch","stepmother","mamma"]],["خود",["own","itself","self","himself","oneself","ego","alter ego","galea","helmet","auto-"]],["را",[]],["ترک",["crack","quit","stop","Turk","abandonment","fracture","renunciation","pillion","refuse","split","craze","cleft","dereliction","chap","desuetude","interstice","proscription","Turkish"]],["کرده،",["relocate","char","do","gig","kick in","perform","ramble","have","joint","rest"]],["به",["to","in","on","into","at","against","quince","quincunx","ac-","ad-","Bah!","Oh!"]],["زن",["woman","female","wife","distaff","carline"]],["خویش",["kindred","relation","connection","connexion","kinswoman","self"]],["خواهد",["want","ask","desire","solicit","wish","require","beg","bone","call","choose","desiderate","intend","invite","invocate","will"]],["پیوست",["annex","appendix","enclosure","annexation","appendage"]],["و",["and"]],["یک",["one","unit","mono-","uni-"]],["تن",["ton","body","person","flesh","corpus"]],["خواهند",["want","ask","desire","solicit","wish","require","beg","bone","call","choose","desiderate","intend","invite","invocate","will"]],["شد.",["become","go","grow","be","happen","leave"]]],[["آدم",[]],["و",["and"]],["زنش",["beat"]],["هر",["any","each","every"]],["دو",["two","twain"]],["عریان",["naked","bare","nude","bald"]],["بودند",["be","stand","exist"]],["و",["and"]],["شرم",["shame","pudency"]],["نداشتند.",["have","possess","own","bear","relieve"]]]]]}}
```

Comment:
Does this need to be keyed to a book (e.g. Genesis)? I'd have thought it could all just be one top level array.

Reply:
Sorry ignore this comment